src/new/helenos/fibril.rs

```diff
@@ -0,0 +1,36 @@
+//! HelenOS fibrils API
+//!
+//! * Header file: <https://github.com/HelenOS/helenos/tree/master/uspace/lib/c/include/fibril.h>
+
+pub use crate::time::*;
+use crate::{
+    c_void,
+    errno_t,
+    size_t,
+};
```

Comment:
Optional nit: `use crate::prelude::*` is a bit preferred to import the ints since most refactoring assumes they are in scope, and it brings a few other useful things.

(Applies to a few of these files)